CARDS-1429: Always display PROMS Scores in the Patient chart

marta- opened the pull request:
* Moved the results section (containing the computed score and interpretation) at the top of the forms after the intro and before the actual survey questions, in a section with `displayType=header`
* Added interpretation computed variable to AUDIT and removed interpretation instructions from intro
* Merged all interpretations in each form under a single computed variable
* Re-labeled the computed variables for consistency across all forms

Includes a small UI improvement to [CARDS-1306](https://phenotips.atlassian.net/browse/CARDS-1306) _Add support for sections that stick to the top (header section) or to the bottom (footer section) of the page when scrolling_: made the appearance of header/footer sections consistent in view and edit mode (stickiness excluded).

marta- committed:
CARDS-1429: Always display PROMS Scores in the Patient chart
* Moved the results section (containing the computed score and interpretation) at the top of the forms after the intro and before the actual survey questions
* Added interpretation computed variable to AUDIT and removed interpretation instructions from intro
* Re-labeled the computed variables for consistency across all forms

sashaandjic commented:
![image](https://user-images.githubusercontent.com/23243244/137376597-c75147f0-30f9-4636-9eb4-040ffe98fe32.png)

sashaandjic commented:
The functionality is all there. Waiting for the final cosmetic changes.

marta- committed:
CARDS-1429: Always display PROMS Scores in the Patient chart
* Improved formatting of AUDIT interpretation
* Cleaned up intro wording in GAD and PHQ
* Moved all interpretation components under the same computed variable in GAD and PHQ

marta- committed:
CARDS-1306: Add support for sections that stick to the top (header section) or to the bottom (footer section) of the page when scrolling
UI improvement: made the appearence consistent in view and edit mode (stickiness excluded)

sashaandjic reviewed (approved):
tested

marta- commented:
> One very minor thing: I think label "Score" and its value are not properly aligned on the patient chart page. Same for the Interpretation.

Reported as https://phenotips.atlassian.net/browse/CARDS-1433. It's a non-blocking general issue (not directly related to 1429).

sdumitriu commented on proms-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/EQ5D.xml:
```diff
@@ -42,6 +42,44 @@
+				<value>computed</value>
+				<type>String</type>
+			</property>
+			<property>
+				<name>displayMode</name>
+				<value>formatted</value>
+				<type>String</type>
+			</property>
```
I don't think this is needed, is it?

marta- replied:
We want it to look like text, not like a disabled input
